engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailProcessor.java

```diff
@@ -37,7 +37,7 @@ public class EmailProcessor extends SystemEndpointTypeProcessor {
     EmailActorsResolver emailActorsResolver;
 
     @Inject
-    EmailSubscriptionTypeProcessor emailSubscriptionTypeProcessor;
+    EmailAggregationProcessor emailSubscriptionTypeProcessor;
```

Comment:
What do you think about refactoring the variable to `emailAggregationProcessor` too?

Reply:
That's right, thank you :)